database/migrations/2017_07_05_102650_add_cover_image_display.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddCoverImageDisplay extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->string('books_display',10)->default('grid');
```

Comment:
- `'books_display'` can we rename it to something better like '**books_view_type**'.
- Is data length 10 is okay for now?